src/Uno.UI.Toolkit/Storage/StorageFileHelper.cs

```diff
@@ -39,5 +51,29 @@ private static Task<bool> FileExistsInPackage(string fileName)
 
 		return Task.FromResult(false);
 	}
+
+	/// <summary>
+	/// Retrieves the paths of assets within the current application based on the specified filter predicate.
+	/// </summary>
+	/// <param name="predicate">A predicate function determining whether a file should be included in the result.</param>
+	/// <returns>Returns an array of strings containing the paths of the filtered assets.</returns>
+	private static Task<string[]> GetFilesInDirectory(Func<string, bool> predicate)
+	{
+		List<string> assetsFiles = new();
+		string path = string.Empty;
+		var executingPath = Assembly.GetExecutingAssembly().Location;
```

Comment:
This is likely going to cause trouble when we'll run as a single file. Let's use the same APIs that:

https://github.com/unoplatform/uno/blob/23feb214275f571018d49295f546994033eb49e3/src/Uno.UWP/Storage/StorageFile.cs#L28

is using.